Dockerfile

```
@@ -1,10 +1,10 @@
-FROM crystallang/crystal:1.10.1-alpine as builder
+FROM crystallang/crystal:1.12-alpine as builder
```

```suggestion
FROM crystallang/crystal:1.14-alpine as builder
```

This should fix the error in https://app.circleci.com/pipelines/github/codacy/codacy-ameba/142/workflows/fd56e448-f53d-4144-95a1-2365b631c925/jobs/511

---

Hello @Sija 
Let me apologize for the huge delay in this reply and thank you for the tips and help with this upgrade.
I'm currently working on bumping Ameba to the latest version in the PR #33 
All packages and versions will be there.
Thanks again. I appreciate your contribution a lot.
David Marinho